docs/resources/cfw_firewall.md

```diff
@@ -0,0 +1,194 @@
+
+  Changing this parameter will create a new resource.
+
+* `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled.
+  Valid values are **true** and **false**. Defaults to **false**.
+
+  Changing this parameter will create a new resource.
+
+<a name="Firewall_Flavor"></a>
+The `flavor` block supports:
+
+* `version` - (Required, String, ForceNew) Specifies the version of the firewall.
+  When the charging_mode is **prePaid**: the value can be **Standard** and **Prefessional**.
+  When the charging_mode is **postPaid**: the value can be **Prefessional**.
+  Changing this parameter will create a new resource.
+
+* `extend_eip_count` - (Optional, Int, ForceNew) Specifies the extend EIP number of the firewall.
+  Only works when the charging_mode is **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `extend_bandwidth` - (Optional, Int, ForceNew) Specifies the extend bandwidth of the firewall.
+  Only works when the charging_mode is **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `extend_vpc_count` - (Optional, Int, ForceNew) Specifies the extend VPC number of the firewall.
+  Only works when the charging_mode is **prePaid**.
+  Changing this parameter will create a new resource.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `engine_type` - The engine type
+
+* `ha_type` - The HA type.
+
+* `protect_objects` - The protect objects list.
+  The [protect_objects](#Firewall_ProtectObject) structure is documented below.
+
+* `service_type` - The service type.
+
+* `status` - The firewall status.
+
+* `support_ipv6` - Whether IPv6 is supported.
+
+* `east_west_firewall_inspection_vpc_id` - The east-west firewall inspection VPC ID.
+
+<a name="Firewall_ProtectObject"></a>
+The `protect_objects` block supports:
+
+* `object_id` - The protected object ID.
+
+* `object_name` - The protected object name.
+
+* `type` - The Project type.
+  The options are as follows: 0: north-south; 1: east-west.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 20 minutes.
+* `delete` - Default is 20 minutes.
+
+## Import
+
+The firewall can be imported using the `id`, e.g.
+
+```bash
+$ terraform import huaweicloud_cfw_firewall.test 6cb1ce47-9990-447e-b071-d167c5393871
+```
+
+Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
+API response, security or some other reason. The missing attributes include:
+`flavor`, `tags`, `period_unit`, `period` and `auto_renew`. It is generally
+recommended running `terraform plan` after importing an CFW firewall. You can then decide if changes should be applied to
+the firewall, or the resource definition should be updated to align with the firewall. Also you can ignore changes as
+below.
+
+```
+resource "huaweicloud_cce_cluster" "cluster_1" {

```

Comment:
```suggestion
resource "huaweicloud_cfw_firewall" "cluster_1" {

```

The following `ignore_changes` should also be changed.

Reply:
fixed